sdp/processors/modify_manifest/data_to_dropbool.py

```diff
@@ -795,3 +797,68 @@ def finalize(self, metrics):
         for insertion, count in total_counter_sorted.items():
             logger.info(f"{insertion}, {count}")
         super().finalize(metrics)
+
+class DropRepeatedFields(BaseParallelProcessor):
+    """Drops utterances from the current manifest if their text fields are present in other manifests.
+
+    This class processes multiple manifest files and removes entries from the current manifest if the text field
+    matches any entry in the other manifests. It allows for optional punctuation removal from the text fields 
+    before performing the check.
+    .. note::
```

Comment:
the note doesn't display properly, I think you need an empty line above this ``.. note::` line